addons/mail/static/src/discuss/core/common/discuss_core_common_service.js

```diff
@@ -31,10 +31,14 @@ export class DiscussCoreCommon {
             const { Thread } = this.store.insert(payload);
             const [thread] = Thread;
             if (thread.notifyOnLeave) {
-                this.notificationService.add(_t("You unsubscribed from %s.", thread.displayName), {
-                    type: "info",
-                });
+                this.notificationService.add(
+                    _t("You left %(channel)s.", { channel: thread.displayName }),
+                    {
+                        type: "info",
+                    }
+                );
```

Comment:
Nitpick: can help prettier a bit to not waste LOCs (thus needing scrolling to navigate in code more efficiently:

```js
this.notificationService.add(
                    _t("You left %(channel)s.", { channel: thread.displayName }),
                    { type: "info" }
                );
```